install.sh

```diff
@@ -1,96 +1,143 @@
 #!/usr/bin/env bash
 set -e
 
-echo Installing protostar
-
-PROTOSTAR_DIR=${PROTOSTAR_DIR-"$HOME/.protostar"}
-mkdir -p "$PROTOSTAR_DIR"
-
-
-PLATFORM="$(uname -s)"
-case $PLATFORM in
-  Linux)
-    PLATFORM="Linux"
-    ;;
-  Darwin)
-    PLATFORM="macOS"
-    ;;
-  *)
-    echo "unsupported platform: $PLATFORM"
-    exit 1
-    ;;
-esac
+PROTOSTAR_REPO="https://github.com/software-mansion/protostar"
+RESULT=""
+
+function create_protostar_directory() {
```

Comment:
```suggestion
create_protostar_directory() {
```

FYI The `function` keyword is unnecessary (really!) :)

Reply:
I know, but VS Code highlights `function` keyword differently. I'll remove it.

Reply:
Removed function keyword